### ISSUE TITLE
Fix HarshHotLifePenalty always stuck on

### ISSUE THREAD

**Issue by andrewipark** [ddrillini/Simply-Love-SM5#19]
This happens across SM 5.0.12 and 5.1 betas, but works on Consensual, so I think the key isn't being updated properly.

**Comment by andrewipark** [comment]
Confirmed for SLv4.8.5.

**Comment by dguzek** [comment]
Sorry about this.  It's probably resolved by https://github.com/dguzek/Simply-Love-SM5/commit/3b8eee136eba880e26a12a4c46c38674917e00b1.

**Comment by andrewipark** [comment]
Appears fixed from casual keyboard testing. (I didn't get around to pulling in upstream until just now.)

**Comment by quietly-turning** [comment]
> I didn't get around to pulling in upstream until just now.

Pending testing, I am hoping very much to have a new SL release out this weekend.

If that occurs and I submit a pull request to this repository, is there a good chance it will be merged in time for RIP12?

**Comment by andrewipark** [comment]
> Pending testing, I am hoping very much to have a new SL release out this weekend.
> 
> If that occurs and I submit a pull request to this repository, is there a good chance it will be merged in time for RIP12?

TL;DR: Yes.

Longer explanation: As is now par for the course, I have *briefly* looked through the subleties of the 4.8.7 commit log. I expect only a few minor downstream revisions on our end.

Also, our cabinet suppliers for RIP12 may or may not choose to use the exact version of our theme as available here for the tournament.

**Comment by quietly-turning** [comment]
> Also, our cabinet suppliers for RIP12 may or may not choose to use the exact version of our theme as available here for the tournament.

This is good to know.  I'd forgotten that the ddrillini machine isn't used during RIP events.

My primary impetus over the last two weeks been making ScreenPlayerOptions more helpful for RIP12.  In that regard, I suppose I'll need to reach out to each cabinet supplier.

**Comment by andrewipark** [comment]
For what it's worth, I am now merging upstream beta in anyways, because the development branch looked fairly stable.

**Comment by quietly-turning** [comment]
> I am hoping very much to have a new SL release out this weekend.

Yeah, I'm bad at this.   Thanks for merging anyway. 👍
